week1/homework-week1/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// 1. String manipulation
+string input = "word and world";

```

Comment:
generally is more common to use `var` instead of specifying the type of the variable